src/proc/layouter.rs

```diff
@@ -73,6 +73,14 @@ impl Layouter {
                                 width: _,
                                 value: crate::ScalarValue::Uint(value),
                             } => value as u32,
+                            // Accept a signed integer size to avoid
```

Comment:
no need to explain it both here and in the validator. Only the latter is source of truth about what's possible.